src/System.Net.NameResolution/src/project.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.0-beta-*",
+    "System.Diagnostics.Contracts": "4.0.0-beta-*",
+    "System.Diagnostics.Tracing": "4.0.0-beta-*",
+    "System.Runtime": "4.0.20-beta-*",
+    "System.Runtime.Extensions": "4.0.10-beta-*",
+    "System.Resources.ResourceManager": "4.0.0-beta-*",
+    "System.Threading": "4.0.0-beta-*",
+    "System.Threading.Tasks": "4.0.0-beta-*",
+    "System.Runtime.InteropServices": "4.0.20-beta-*",
+    "System.Runtime.Handles": "4.0.0-beta-*",
+    "System.IO": "4.0.10-beta-*",
+    "System.IO.FileSystem": "4.0.0-beta-*",
+    "System.IO.FileSystem.Primitives": "4.0.0-beta-*",
+    "System.ComponentModel.EventBasedAsync": "4.0.10-beta-*",
+    "System.Globalization": "4.0.0-beta-*",
+    "System.Diagnostics.Debug": "4.0.10-beta-*",
+    "System.Collections.NonGeneric": "4.0.0-beta-*",
+    "System.Security.Claims": "4.0.0-beta-*",
+    "System.Security.SecureString": "4.0.0-beta-*",
+    "Microsoft.Win32.Primitives": "4.0.0-beta-*",
+
+    "System.Threading.ThreadPool": "4.0.10-beta-*",
+    "System.Threading.Overlapped": "4.0.0-beta-*",
+    "System.Security.Principal.Windows": "4.0.0-beta-*",
+
+    "System.Net.Primitives": "4.0.10-beta-*"
+  },
+  "frameworks": {
+    "dnxcore50": {}
+  }
+}
```

Comment:
You may have already done this, but I believe these should be updated at this point to use stable versions where possible.